src/latency.h

```diff
@@ -89,4 +91,34 @@ void latencyAddSample(const char *event, mstime_t latency);
 #define latencyRemoveNestedEvent(event_var,nested_var) \
     event_var += nested_var;
 
+struct durationStats {
+    long long cnt;
+    long long sum;
+    long long max;
+};
+
+void durationAddSample(const char* event, ustime_t duration, int add_to_latency);
```

Comment:
There are many more places below.
```suggestion
void durationAddSample(const char *event, ustime_t duration, int add_to_latency);
```